schematic/store/synapse.py

```diff
@@ -202,12 +203,14 @@ def __init__(
         token: Optional[str] = None,  # optional parameter retrieved from browser cookie
         access_token: Optional[str] = None,
         project_scope: Optional[list] = None,
+        synapse_cache_path: str = ".synapseCache",
```

Comment:
This will change the current default behavior. By default, synapse python client set the cache directory in root. For example, for me, the cache directory is: `/Users/lpeng/.synapseCache`. But here, if you set synapse_cache_path default as `.synapseCache`, then it will change the cache directory as the same folder as I am running schematic. The change of `.synapseCache` directory doesn't really matter for me, but I think it might impact external users. I don't really see a compelling reason to change the default cache directory (I think our goal is just to make the .synapseCache folder location configuration). Can we observe the current behavior?